web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -169,7 +169,7 @@
                 </span>
             </div>
         </div>
-        <div v-if="!isNewAccessGrantFlow">
```

Comment:
why is this flag flipped? With this change, the old access flow no longer properly lists accesses. This is the only comment blocking my approval. Everything else looks good to me, and I'm happy to approve once the old access grants flow works on this branch

Reply:
fixed